app/src/main/res/layout/activity_login.xml

```diff
@@ -2,92 +2,107 @@
 <LinearLayout
```

Comment:
can we check and remove unnecessary attributes if any in this layout

Reply:
I don't feel there are any unnecessary attributes in this layout.